vue.config.js

```diff
@@ -3,10 +3,10 @@ const PrerenderSPAPlugin = require('prerender-spa-plugin');
 const path = require('path');
 module.exports = {
     // was working but now not working why?
```

Comment:
if working then please remove this comment

Reply:
done